pyiso/pjm.py

```diff
@@ -60,11 +81,55 @@ def fetch_edata_series(self, data_type, params=None):
 
         # parse html to df
         dfs = pd.read_html(response.content, header=0, index_col=0, parse_dates=True)
-        df = self.utcify_index(dfs[0])
+        df = self.utcify_index(dfs[0], tz_name='utc')
```

Comment:
this change breaks timezone handling for me, maybe it was working for you Nate because you're in the same local time as PJM?